OpenRA.Mods.Common/Traits/Render/SupportPowerChargeBar.cs

```diff
@@ -36,13 +37,20 @@ public SupportPowerChargeBar(Actor self, SupportPowerChargeBarInfo info)
 
 		float ISelectionBar.GetValue()
 		{
-			if (!self.Owner.IsAlliedWith(self.World.RenderPlayer))
-				return 0;
-
 			var spm = self.Owner.PlayerActor.Trait<SupportPowerManager>();
```

Comment:
Saw that too, but didn't want to touch that in this pr. Can do it ofc and will probably,
now that someone explicitly asked.